Installation/include/CGAL/license/generate_files.cmake

```
@@ -5,6 +5,10 @@ foreach(LINE ${package_list})
   string(REPLACE "/" "_" PACKAGE_CODE_NAME ${PACKAGE_NAME})
   string(TOUPPER ${PACKAGE_CODE_NAME} PACKAGE_CODE_NAME)
   string(REPLACE "${PACKAGE_NAME}" "" PACKAGE_PRETTY_NAME ${LINE})
+  string(COMPARE EQUAL ${PACKAGE_PRETTY_NAME} " " IS_EMPTY)
+  if(${IS_EMPTY})
+    set(PACKAGE_PRETTY_NAME ${PACKAGE_NAME})
+  endif()
```

As an empty variable evaluates to false in conditional, I suggest this more idiomatic construction:
```suggestion
  if(NOT PACKAGE_PRETTY_NAME)
    set(PACKAGE_PRETTY_NAME ${PACKAGE_NAME})
  endif()
```
Please test if that works as well as the original patch.

---

I slightly modified your suggestion because `PRETTY_PACKAGE_NAME` evaluates to "one whitespace", which does not evaluate to false. However, after stripping all trailing whitespaces it does work. I have also tested it locally and it seems to work well (the original patch I also tested and it is also valid).